compiler/control/CompileMethod.cpp

```diff
@@ -303,6 +304,7 @@ compileMethodFromDetails(
          {
          TR_VerboseLog::writeLine(TR_Vlog_INFO, "%s cannot be translated", compilee.signature(&trMemory));
          }
+      TR_VerboseLog::vlogRelease();
```

Comment:
Recommend changing `writeLine` to `writeLineLocked` since there are multiple exit points.

Reply:
Agreed